x11-libs/libpciaccess/libpciaccess-0.14.recipe

```diff
@@ -1,32 +1,42 @@
 SUMMARY="Standardized interface for PCI access"
 DESCRIPTION="Standardized interface for accessing PCI devices."
-HOMEPAGE="http://xcb.freedesktop.org/"
+HOMEPAGE="https://www.x.org/releases/individual/lib/"
 COPYRIGHT="2006, 2007 IBM Corporation
 	2006 Eric Anholt
 	2011 Mark Kettenis
 	2012 Robert Millan
 	2007, 2008, 2009, 2011, 2012, 2013 Oracle and/or its affiliates.
-	2009, 2012 Red Hat, Inc."
+	2009, 2012 Red Hat, Inc.
+	2007 Paulo R. Zanoni, Tiago Vignatti
+	2009 Tiago Vignatti
+	2000 The XFree86 Project, Inc.
+	2009, 2012 Samuel Thibault
+	2009 Michael Lorenz
+	2008, 2011 Mark Kettenis
+	2008 Juan Romero Pardines"
 LICENSE="MIT"
 REVISION="1"
-SOURCE_URI="https://xorg.freedesktop.org/releases/individual/lib/libpciaccess-$portVersion.tar.gz"
-CHECKSUM_SHA256="fe26ec788732b4ef60b550f2d3fa51c605d27f646e18ecec878f061807a3526e"
+SOURCE_URI="https://www.x.org/releases/individual/lib/libpciaccess-$portVersion.tar.bz2"
+CHECKSUM_SHA256="3df543e12afd41fea8eac817e48cbfde5aed8817b81670a4e9e493bb2f5bf2a4"
 PATCHES="libpciaccess-$portVersion.patchset"
 
-ARCHITECTURES="?x86_gcc2 ?x86 ?x86_64"
-SECONDARY_ARCHITECTURES="?x86"
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
```

Comment:
@Begasus, if you feel like writing a patch for fixing an error at line 230 of `scanpci/scanpci.c`, you can mark x86_gcc2 as tested :)
```diff
-ARCHITECTURES="?x86_gcc2 x86 x86_64"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
```
Here's the diff for the patch:
```diff
     ret = pci_system_init();
     if (ret != 0)
-	err(1, "Couldn't initialize PCI system");
+	err(1, "%s", "Couldn't initialize PCI system");
```

I'm not sure, however, if the correct format to use is just `"%s"` or if we should use `"%s\n"`. I'll leave that to you ;-)

BTW, everything else looks OK in your recipe, and I'll test it soon on x86_64.

Reply:
Shall we use a PATCH() function?
```
PATCH()
{

	if [ "$effectiveTargetArchitecture" = x86_gcc2 ]; then
		sed -i -e "s/\(err(1, \)\(\"Couldn't initialize PCI system\");\)/\1\"%s\", \2/;" \
			scanpci/scanpci.c
	fi
}
```
Or would you prefer to mark this recipe as broken on x86_gcc2?

Reply:
Insterted it as a PATCH function, easier to remove is we would like to set x86_gcc2 as broken, it builds ok, but there is no need for it as other deppendencies that would call it won't work with gcc2*